.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
I think there's a question here over whether we want to leave our repos as editor agnostic. For example, me and others on the team don't use VS Code. There is precedent in Hackney for not committing the`.vscode` folder to the repo.

Reply:
Understood and addressed in [815846d](https://github.com/LBHackney-IT/evidence-api/pull/161/commits/815846da988ac93829aebd10c15d568dc532c78f)